README.md

```diff
@@ -6,14 +6,14 @@ A repository that includes Hello World from popular languages added by our commu
 
 1. Fork the repository, and create a branch named `<language_to_add>`. In this example, we're adding **Java** to the list.
 2. Create a file in the `lang` folder with the name of the language you want to add (e.g.: `java.md`).
-3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backticks.
+3. Add this content to your newly created file. Add your GitHub username in the `author` field. Just remove the `\` in the code backtick.
 
 ```markdown
 ---
 lang: Java
 syntax: java
 title: Java is so cool!
-author: Panquesito7
+author: DEVSTRONS
```

Comment:
I used `Panquesito7` here as an example of myself adding the Java language, so you should probably revert it or add another username.